addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2116,7 +2151,7 @@ we-select.o_we_border_preview_aligned_select {
         transition: opacity 400ms linear 0s;
 
         &.o_overlay_hidden {
-            opacity: 0;
+            opacity: 0 !important;
```

Comment:
Without it, you'll get an always-visible overlay on scroll for selected text highlights, since we reduce it (for more highlight visibility on small text) in the `TextHighlight` option.

Reply:
> TextHighlight

maybe you could use the `opacity-25` class on the overlay

Reply:
> Without it, you'll get an always-visible overlay on scroll for selected text highlights, since we reduce it (for more highlight visibility on small text) in the `TextHighlight` option.

maybe you could use the `opacity-25` class on the overlay